Makefile

```diff
@@ -40,33 +47,35 @@ start:
 dev-server: CONFIG_ENV = development
 dev-server: CALYPSO_ENV = desktop-development
 dev-server: NODE_ENV = development
-dev-server: checks
+dev-server: checks desktop$/config.json
 	@echo "\n\n$(GREEN)+------------------------------------------------+"
 	@echo "|                                                |"
 	@echo "|    Wait for calypso to start the dev server    |"
 	@echo "|       and start the app with \`make dev\`        |"
 	@echo "|                                                |"
 	@echo "+------------------------------------------------+$(RESET)\n\n"
-	
-	@$(MAKE) build-config CONFIG_ENV=$(CONFIG_ENV)
-
 
 	@npx concurrently -k \
 	-n "Calypso,Desktop" \
 	"$(MAKE) calypso-dev NODE_ENV=$(NODE_ENV) CALYPSO_ENV=$(CALYPSO_ENV)" \
-	"wait-on http://localhost:3000 && $(MAKE) desktop-dev NODE_ENV=$(NODE_ENV)" \
+	"wait-on http://localhost:3000 && $(MAKE) build-desktop NODE_ENV=$(NODE_ENV)" \
 
 # Start app in dev mode
 dev: NODE_ENV = development
 dev: DEBUG = desktop:*
 dev: 
 	$(MAKE) start NODE_ENV=$(NODE_ENV) DEBUG=$(DEBUG)
 
-# Build config
-build-config: 
-	@node $(THIS_DIR)/resource/build-scripts/build-config-file.js $(CONFIG_ENV)
+desktop-config$/config-%.json:
+	$(info Config file for environment "$(CONFIG_ENV)" does not exist. Ignoring Environment.)
```

Comment:
this one may have a bug and I know I suggested the code which means I introduced it.

this will run whenever the config file changes as well as when it's missing. this might be
a good time to invoke a shell to confirm the presence of the file. I think the following
should work (but it's probably wrong)

```Makefile
desktop-config$/config-%.json:
	@if [ -z $@ ]; then echo "Config file for environment \"$(CONFIG_ENV)\" does not exist. Ignoring environment."; fi
```

Reply:
For simplicity I merged that with `desktop-config/config.json` as it failed in the JS liner as well.